scripts/pre_commit_linter.py

```diff
@@ -376,12 +377,19 @@ def _lint_py_files(config_pylint, files_to_lint, result):
         print 'Linting Python files %s to %s...' % (
             current_batch_start_index + 1, current_batch_end_index)
 
-        try:
-            # This prints output to the console.
-            lint.Run(current_files_to_lint + [config_pylint])
-        except SystemExit as e:
-            if str(e) != '0':
-                are_there_errors = True
+        # These lines print output to the console.
```

Comment:
Not clear from comment  what "these lines" refer to, especially since the next line is a comment.

Reply:
I have broken the comment for Pylint and Pycodestyle to make it clear. Done!